.kitchen.yml

```diff
@@ -5,23 +5,37 @@ driver:
 provisioner:
   name: chef_zero
   chef_client_path: /usr/bin/chef-client
-  always_update_cookbooks: true
-  multiple_converge: 2
-  enforce_idempotency: true
+  client_rb:
+    chef_license: accept
+    local_mode: true
+  attributes:
+    chef_license: accept
+    effortless:
+      origin: ncerny
+      pkg: infra-client
```

Comment:
Apologies if you told me about this before and I forgot - could we not be using chef/chef-client here?

Reply:
It needs this PR merged before we can use it:  https://github.com/chef/chef/pull/8598

It will break existing customers as it is today.